### PR TITLE
fix #79525 faupto.com

### LINKED ISSUE #79525: faupto.com
### Issue URL (Ads)
 [https://faupto.com/sldoge/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ffaupto.com%2Fsldoge%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880), <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard Russian, <br/>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by wout-codes**
#79525

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -261,6 +261,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+faupto.com###ads-hJiok
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt:264]: second part is random. 
Also can be blocked 
`||dogemate.com/assets/img/*_728x90.`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:264]: Where is the second part? I don't see?

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:264]: Does this work? `faupto.com##div[id^="ads-"]` or `faupto.com##div[id*="ads-"]`

Comment (Alex-302) [reply EnglishFilter/sections/specific.txt:264]: better `faupto.com##div[id^="ads-"]`

Comment (Alex-302) [reply EnglishFilter/sections/specific.txt:264]: > Where is the second part?

`hJiok`)

Comment (Alex-302) [reply EnglishFilter/sections/specific.txt:264]: Ok, will change in master.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**